src/Accounts/Authentication/AzureSessionInitializer.cs

```diff
@@ -202,6 +211,18 @@ static ContextAutosaveSettings InitializeSessionSettings(IDataStore store, strin
             return result;
         }
 
+        static String GetAzureCLIInstallationId(){
+            String installationId = null;
+            if (File.Exists(AzProfileInfoFile))
```

Comment:
Always use the IDataStore abstraction
```suggestion
        static String GetAzureCLIInstallationId(IDataStore store) {
            String installationId = null;
            if (store.FileExists(AzProfileInfoFile))
```